admin/services/data-access/pupil.data.service.js

```diff
@@ -85,10 +85,13 @@ pupilDataService.sqlFindOneBySlug = async function (urlSlug, schoolId) {
   ]
   const sql = `
       SELECT TOP 1 
-      *  
-      FROM ${sqlService.adminSchema}.${table}
-      WHERE urlSlug = @urlSlug
-      AND school_id = @schoolId  
+      p.*,
+      pag.reason AS ageReason
+      FROM ${sqlService.adminSchema}.${table} p
+      LEFT OUTER JOIN ${sqlService.adminSchema}.[pupilAgeReason] pag
```

Comment:
Why would we return the pupilAgeReason in a generic method used in many places?

Reply:
good catch.  this should be changed, as the main driver for a separate table was the overhead

Reply:
@GuyHarwood @jon-shipley it's changed now.